ContextDesktop/src/main/java/com/github/migi_1/Context/model/MainEnvironment.java

```diff
@@ -576,4 +578,13 @@ public BoundingBox getLeftBound() {
     public BoundingBox getRightBound() {
         return boundingBoxWallRight;
     }
+
+    /**
+     * @param app the app to set
+     */
```

Comment:
Doesn't Checkstyle complain about this Javadoc?
It doesn't contain a description (that has to end with a dot).
Just a parameter description is not enough.